test/resources/OAuth.spec.js

```diff
@@ -0,0 +1,177 @@
+'use strict';
+
+var stripe = require('../../testUtils').getSpyableStripe();
+
+var expect = require('chai').expect;
+
+describe('OAuth', function() {
+  describe('authorize', function() {
+    describe('when a default client_id is not set', function() {
+      beforeEach(function() {
+        stripe.setClientId('');
```

Comment:
I think the default resting state of the world is for it to be `undefined`; should we `stripe.setClientId()` instead?